tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,421 @@
+(e.g., rotation curves for non-spherical objects, outgassing events).
+All changes in the brightness of DP0.3 objects with time are due to changes in the distance and phase angle from Earth.  
+
+4.1. Execute the following ADQL query to retrieve the r-band magnitudes, phase angles,
+heliocentric and topocentric distances, and time of the observations for the TNO explored in Step 3.
+
+.. code-block:: SQL 
+
+    SELECT ds.midPointMjdTai, ds.mag, ds.band, 
+    ss.phaseAngle, ss.topocentricDist, ss.heliocentricDist 
+    FROM dp03_catalogs_10yr.DiaSource AS ds 
+    JOIN dp03_catalogs_10yr.SSSource AS ss ON ds.diaSourceId = ss.diaSourceId
+    WHERE ss.ssObjectId = -735085100561880491
+    AND ds.band = 'r'
+
+4.2. The default plot will have the r-band magnitude as a function of time.  
+Use the plot "Settings" function to add new scatter plots showing the magnitude and the phase angle as a function of time, ``midPointMjdTai - 60000``.  
+This will result in the left two plots, as on the screenshot below.   Note that these quantities are not correlated with time.
+Add a new scatter plot showing the r-band magnitude as a function of phase angle (right plot), showing that the phase angle and r-band magnitude are correlated.
+
+.. figure:: /_static/portal_tut03_step04a.png
+    :name: portal_tut03_step04a
+    :width: 600
+    :alt: A screenshot of three plots showing magnitude and phase angle are not correlated with time, and that magnitude is correlated with phase angle.
+
+    Three plots demonstrating that magnitude and phase angle are correlated with each other, but not with time.
+
+4.3.  Plot the topocentric and heliocentric distances of the object as a function of time.  
+Here, use the columns in the table which you generated in Step 4.1.  
+First, delete two of the the three plots prepared in Step 4.2 by clicking on the blue ``X`` in the upper right-hand part of the plot panels to make space for new plots.  
+Then add a pair of new plots, clicking on the "plot settings."  
+In both cases, enter ``midPointMjdTai - 60000`` for X-axis, but for Y axis - enter ``topocentricDist`` for one plot, and ``heliocentricDist`` for the other.  
+After you remove the panel containing the plot made in the previous step, you will see the plots as below.  
+
+.. figure:: /_static/portal_tut03_step04b.png
+    :width: 600
+    :name: portal_tut03_step04b
+    :alt: A screenshot of two plots showing the heliocentric and topocentric distance of the trans-Neptunian object as a function of time.
+
+    Heliocentric and topocentric distance of the TNO as a function of time.  
+Note the periodic change of the topocentric distance with time resulting from the Earth's motion around the Sun - a different view of the same effect you saw in Step 3.5.  
+
+.. _DP0-3-Portal-3-Step-5:
+
+Step 5. Visualize in 3-D the trajectory of the TNO by viewing the 2-D projection of its orbit
```

Comment:
This title got pretty long, shorter is better, to make the ToC (at upper right in the rendered version) more navigable.

Reply:
OK, if needed it can be made even shorter than my current attempt.